src/components/molecules/ChatList/ChatMessage.tsx

```diff
@@ -55,33 +50,66 @@ export const ChatMessage: FC<ChatMessageProps> = ({
     onDeleteClick(message);
   }, [message, onDeleteClick]);
 
-  return message.from in usersById ? (
+  if (!user || !profile) return null;
+
+  const sender = { ...usersById[message.from], id: message.from };
+  const isMe = message.from === user?.uid;
```

Comment:
[nit] Since we check `!user` above, do we need to null check `user?.uid` here?
```suggestion
  const isMe = message.from === user.uid;
```

Reply:
https://github.com/sparkletown/sparkle/pull/921/commits/f69bd1a816115991a5ca4257830f370237e55697